crates/forge/bin/cmd/create.rs

```diff
@@ -537,7 +538,7 @@ impl CreateArgs {
                     .constructor()
                     .ok_or_else(|| eyre::eyre!("could not find constructor"))?
                     .abi_encode_input(&args)?;
-                constructor_args = Some(hex::encode(encoded_args));
+                constructor_args = Some(hex::encode_prefixed(encoded_args));
```

Comment:
Wouldn't this change for normal evm verification as well? If yes, is the `0x` handled fine for EVM but not on zkSync?

Reply:
So on zksync.io the prefix needs to be there, but Etherscan handles it with or without. Later in the verification method the prefix is stripped anyways, so I added it here for consistency, considering that otherwise it's verbatim what you give to `--constructor-args`, and if you get it from `cast abi-encode` it's prefixed